### ISSUE TITLE
Schema/table name replacement 

### ISSUE THREAD

**Issue by elewis787** [VauntDev/tqla#17]
Currently - it is not possible to replace the schema name dynamically without doing a string replace outside of tqla. 

This is largely due to the placeholder replace logic. 

To support this, we will likely need to add internal functions or node objects to identify a template node and how to handle it differently - i.e - should we replace with a placeholder ( currently how all replaces work ) or string replace. 

The challenges will be related to still ensuring proper SQL prevention on string replacement.

**Comment by enverbisevac** [comment]
Yep, at the moment no other way `FROM <schema>.articles` and I am replacing it after query is compiled. It would be nice to have this feature.

**Comment by elewis787** [comment]
related to #25
